### DISCUSSION TITLE
Clickable UI for tool options (suggested by u/Moratamor on Reddit)

### DISCUSSION THREAD

**Discussion by katepolak** [voxeltycoon/issues#476] [General]
**Is your feature request related to a problem? Please describe.**
Currently there is no way to change parameters of a tool other than hot keys, apparently some people prefer to use the mouse only,

**Describe the solution you'd like**
My idea on how to fix this would also fix the issue of people not noticing the hints on the bottom right. Basically changing it to a window with background so it's more noticeable and making the key images clickable.

**Describe alternatives you've considered**
Rework it entirely. But I don't think that would be exactly a great idea.

**Additional context**
https://www.reddit.com/r/voxeltycoon/comments/c7at82/can_we_get_ui_for_station_lengthwidth_etc/

**Comment by Zeragamba** [comment]
I just bought the game, and I failed to notice the keybinds down in the bottom right of my screen, and so I often had stations next to each other not realizing I could add more tracks to a station. Having a GUI based buttons that controlled aspects of the station (passenger/freight, length, tracks) would make it easier for players who don't want to rely on keybinds.

An example could be:
![station GUI mock](https://user-images.githubusercontent.com/2325473/60402849-1c471980-9b63-11e9-909d-f126bd9795b9.png)

**Comment by Zeragamba** [comment]
This would also be good for quickly jumping to what kind of rail signal to use (type: normal, chained; style: vanilla, mod1, mod2). I often find myself cycling through all the options, and accidentally skipping over the signal i wanted forcing me to loop through all of them again

**Comment by andrewpey** [comment]
Yeah, we plan to have UI for all tools, with hotkeys-only approach only as a temporary solution.

We already added UI for signal type picking in `0.73`, and windows for other tools are coming in the future versions.
